src/Input/Silk.NET.Input.Sdl/SdlKeyboard.Keys.cs

```diff
@@ -0,0 +1,260 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System.Collections.Generic;
+using Silk.NET.SDL;
+
+namespace Silk.NET.Input.Sdl
+{
+    partial class SdlKeyboard
+    {
+        private static readonly Dictionary<KeyCode, Key> _keyMap = new Dictionary<KeyCode, Key>
+        {
+            // The key enum was designed for UK-style QWERTY keyboards, thus some enumerants are missing if they require
+            // a modifier key (such as a shift) in order to access that key
+
+            {KeyCode.KUnknown, Key.Unknown},
+            {KeyCode.KReturn, Key.Enter},
+            {KeyCode.KEscape, Key.Escape},
+            {KeyCode.KBackspace, Key.Backspace},
+            {KeyCode.KTab, Key.Tab},
+            {KeyCode.KSpace, Key.Space},
+            //{KeyCode.KExclaim, Key.Exclaim},
+            //{KeyCode.KQuotedbl, Key.Quotedbl},
+            //{KeyCode.KHash, Key.Hash},
```

Comment:
I have absolutely no clue what's going on here, why are some commented? what does UK-style QWERTY have to do with it?

Reply:
Some are commented if they don't have a corresponding Silk.NET.Windowing.Key enumerant. Instead of just removing them, I left them commented out if such a time comes that they are added.